src/components/Questionnaire/CodingEditor.tsx

```diff
@@ -55,14 +58,16 @@ export function CodingEditor({ code, onChange }: CodingEditorProps) {
           size="sm"
           onClick={() => {
             onChange({
-              system: Object.values(TERMINOLOGY_SYSTEMS)[0],
+              system: isUnit
+                ? Object.values(TERMINOLOGY_SYSTEMS)[2]
+                : Object.values(TERMINOLOGY_SYSTEMS)[0],
```

Comment:
Why not? 
```suggestion
                ? TERMINOLOGY_SYSTEMS.UCUM
                : TERMINOLOGY_SYSTEMS.LOINC,
```

This way, if some changes the order of the keys it will not break and the possibility of changing key name is very less

Reply:
Yup we can do that